tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_change_language.py

```diff
@@ -10,17 +10,20 @@
 
 class TestChangeLanguage(GaiaTestCase):
 
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.data_layer.set_setting('devtools.pseudolocalization.enabled', True)
+
     def test_change_language_settings(self):
         lang_name = self.marionette.execute_script("""
-            var  = window.wrappedJSObject.navigator.mozL10n.;
-            return ['fr-x-psaccent'].translate('Packaged Accented');
+            var qps = window.wrappedJSObject.navigator.mozL10n.qps;
```

Comment:
In l20n.js, we removed the `mozL10n.qps` API and called it: `document.l10n.pseudo`.  The Settings app doesn't use l20n yet, however, and shared/js/l10n.js still uses "qps" as the name of the property.